src/components/PasswordStrengthMeter/index.tsx

```diff
@@ -93,11 +93,15 @@ const PasswordStrengthMeter = ({
 
   return (
     <div title={title} role="alert" aria-live="polite">
-      <StyledTitle variant="bodyB" color={theme.colors.neutral.text}>
+      <StyledTitle variant="bodySmallStrong" as="h3">
         {title}
       </StyledTitle>
 
-      <StyledStrength as="span" variant="bodyB" color={strength[score]?.color}>
+      <StyledStrength
+        as="span"
+        variant="bodySmallStrong"
+        strength={strength[score]}
+      >
```

Comment:
Here too?

Reply:
You mean `as="span"`? Currently the strength score is a span element

Reply:
@baptisteArno I mean here too it should be a `bodySmall` instead of `bodySmallStrong` as shown in migration doc:

> bodyB → bodySmall

Reply:
Check `StyledStrength`, I removed the font-weight props, and setting the `Strong` suffix instead. Is it how we are supposed to do it?